### ISSUE TITLE
Consent & Opt-in re: decision-making at the time of diagnosis and disadvantaged social groups 

### ISSUE THREAD

**Issue by BierJess** [DP-3T/documents#95]
Under section 3, Handing infected patients, the details of opting in should be handled with great care especially if done at the time of diagnosis rather than, say, at the time that the patient is tested (but before the results come back). The user should provide informed consent before opting in, stating that they understand the basic goals, background, and methods of the study and its handling of data, personal or otherwise. This should not just be a popup in the app (thought it can also be that), but also a printed out 1-2 page sheet summarizing the reasoning behind opting in and the use of data. App consent tick boxes tend to be ignored by users as many of them are dozens of pages long, and so although these are legally sufficient in some cases for obtaining consent, they are not considered ethically sufficient by some qualitative researchers and relevant university ethics review boards. There should be consent forms provied in multiple relevant languages as necessary, and these should also be freely available online. 

Keep in mind that the request may be made at an incredibly difficult personal and financial time for the patient, who would have just received a difficult and unpredictable diagnosis that may require complete isolation from family and friends, that the healthcare professionals are likely to be rushed and overworked and so to forget or (out of practical necessity) try to fast track consent and cutt corners, and that the patient may feel disempowered with respect to authority figures such as their healthcare provider. 

Regarding disadvantaged social groups: in general for the app, not just those who test positive, there should also be clear suggested cautionariy advice in terms of who is able to opt in and these should be very specific, excluding e.g. all those under the age of 18, those who are part of groups who may face extra risks (undocumented people, those with migration status concerns, homeless people, those in domestic violence situations where their partner may ultimately have access to their phone to view their risk factor, and those on probation or with felony criminal records, and any who is for other reasons are not in a good position to make such a decision.)

**Comment by pdehaye** [comment]
Thank you for raising this issue, @BierJess. This is also something I worry about. 

I think highlighting [this example with Uber drivers in Mexico taking a Chinese patient can be helpful](https://www.reuters.com/article/us-china-health-mexico/uber-suspends-240-accounts-in-mexico-to-prevent-coronavirus-spreading-idUSKBN1ZW003) can be helpful to think through some of the issues you are raising.  

We have to keep in mind that Uber had God's view over the contacts in this case (except for potential passengers) is _better_ from a contact tracing standpoint than DP-3T/Apple/Google could afford as a contact tracing system. Indeed, Uber has an exact view of time, while DP-3T-protocol'd apps won't. When the taxi driver will activate the signalling that he is infected, he will do so for all his passengers in the last 14 days by default. How will he be given advice on what time period to trigger for? 

DP-3T will collapse contexts like that a lot. From my understanding of what is technically feasible, people will just get notifications, without any contextualization of the likelihood that this ping is significant to them (by design). Uber drivers for instance also have families, go to the shop, etc. So a ping will be felt very differently by all, and - this seems to have been missed by many - the significance of those pings will be vastly increased if friends/relatives/neighbors do have the app installed as well and are able to tell them "I probably triggered your ping". Not baking in this type of information in the protocol will push offline the search for such information, which might be dangerous. 

Note that the significance of these pings to individuals will harmonize over time as we fight back the epidemic (fewer and fewer pings, more and more pressure to at least test yourself), but I would expect it to have very different significance to all of us as we get deconfined.

**Comment by zukunft** [comment]
I agree with simple explaining popups in case of handing infected patients. This is an exceptional case, where nothing should go wrong. I guess in this case most user would take there time to fill in the dialog, 
Optional the time that the patient has been tested or even the time of potential infection or time of symptoms start should be added to the confirmation dialog. This additional information could improve the suggested time of testing of the contacts. Assume you get a message, that you have been a contact of someone who has now been tested positive. Probably one of the first things that you want to know is, when would be the best time for you to be tested. Because there is no clear answer, you would like to have as much information as possible. I know of doctors who have tested patients several times until they get a positive test result.
